Update jdk15 to jdk16

https://github.com/AdoptOpenJDK/build-jdk/blob/master/.github/workflows/nightly.yml#L13

A couple of issues:

1. we can support jdk16, i.e. if jdk16u is not available.  The cons is we might need to constantly switch between ` javaToBuild :jdk** ` and ` javaToBuild :jdk**u` in the nightly workflow file. 
2. jdk15+ get error like  `configure: error: Invalid value for --enable-jvm-feature-dtrace: 'auto'`,  https://github.com/AdoptOpenJDK/build-jdk/runs/1765281642?check_suite_focus=true .  Looks like from jdk15 we need to switch from `--enable-dtrace=auto` to `--enable-dtrace` https://github.com/openjdk/jdk/blob/master/make/autoconf/jvm-features.m4#L99.   Looks like openjdk-build set another layer to do those configuration `openjdk-build/build-farm/platform-specific-configurations`. For build-jdk we actually doesn't use this layer. If we use this layer the story will be different from developer's local story. If we don't use this layer we might need to continually do extra job to deal with those configuration.

--- a/.github/workflows/nightly.yml
+++ b/.github/workflows/nightly.yml
@@ -10,12 +10,12 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-16.04, macos-10.15, windows-2016]
-        javaToBuild: [jdk11u, jdk15u]
+        javaToBuild: [jdk11u, jdk16u]
     steps:
     - uses: actions/checkout@v1
     - uses: AdoptOpenJDK/build-jdk@v1
       id: buildJDK
-      with: 
+      with:
         javaToBuild: ${{ matrix.javaToBuild }}
     - run: ${{ steps.buildJDK.outputs.BuildJDKDir }}/bin/java -version
   openj9:
@@ -24,12 +24,12 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-16.04, macos-10.15]
-        javaToBuild: [jdk11u, jdk15u]
+        javaToBuild: [jdk11u, jdk16u]
     steps:
     - uses: actions/checkout@v1
     - uses: AdoptOpenJDK/build-jdk@v1
       id: buildJDK
-      with: 
+      with:
         javaToBuild: ${{ matrix.javaToBuild }}
         impl: 'openj9'
     - run: ${{ steps.buildJDK.outputs.BuildJDKDir }}/bin/java -version

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -11,12 +11,12 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-16.04, macos-10.15, windows-2016]
-        javaToBuild: [jdk11u, jdk15u]
+        javaToBuild: [jdk11u, jdk16u]
     steps:
     - uses: actions/checkout@v1
     - uses: ./
       id: buildJDK
-      with: 
+      with:
         javaToBuild: ${{ matrix.javaToBuild}}
     - run: ${{ steps.buildJDK.outputs.BuildJDKDir }}/bin/java -version
   openj9:
@@ -25,12 +25,12 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-16.04, macos-10.15]
-        javaToBuild: [jdk11u, jdk15u]
+        javaToBuild: [jdk11u, jdk16u]
     steps:
     - uses: actions/checkout@v1
     - uses: ./
       id: buildJDK
-      with: 
+      with:
         javaToBuild: ${{ matrix.javaToBuild}}
         impl: 'openj9'
     - run: ${{ steps.buildJDK.outputs.BuildJDKDir }}/bin/java -version

